### DISCUSSION TITLE
What are the system requirements of revolt?

### DISCUSSION THREAD

**Discussion by ffuentese** [stoatchat/discussions#359] [General]
I know this is a work in progress but I look at the website, the main repositories and can't find system requirements, especially Ram and CPU. Let's say I wanna have a server for me and my friends. Can anyone make this work on a Raspberry Pi 4 maybe?

**Comment by FanMclaine** [comment]
> Can anyone make this work on a Raspberry Pi 4 maybe?

Discord can run on a raspberry pi 4 with 8gb of RAM and i think same with the 4gb one. So yes, you can use Revolt on a pi. If your pi struggled to load Revolt, then you should use [Retaped](https://error-404-null-not-found.github.io/Retaped/)

**Reply by ffuentese** [reply]
The link is not found.

**Reply by FanMclaine** [reply]
> The link is not found.

try this https://error-404-null-not-found.github.io/Retaped/
